lib/components/Input/base/index.tsx

```diff
@@ -252,25 +263,40 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
                     {showInputButton && InputButton}
                 </div>
-                {(message || showCharacterCounter) && !hideMessage && (
-                    <div
-                        className={clsx(
-                            "message__container",
-                            `message__container--${inputSize}`,
-                            `message__container__status--${status}`,
-                            disabled && `message__container__disabled`,
-                        )}
-                    >
-                        <span className="message__container__text">
-                            {message}
-                        </span>
-                        {showCharacterCounter && maxLength && (
+                {(message ||
+                    showCharacterCounter ||
+                    hasPasswordStrengthValidation) &&
+                    !hideMessage && (
+                        <div
+                            className={clsx(
+                                "message__container",
+                                `message__container--${inputSize}`,
+                                `message__container__status--${status}`,
+                                disabled && `message__container__disabled`,
+                            )}
+                        >
+                            {hasPasswordStrengthValidation && (
+                                <div className="message__container__password_validation">
+                                    {validationMessages?.map((msg, idx) => (
+                                        <PasswordStrengthValidation
+                                            key={idx}
+                                            status={status}
+                                            validationMessage={msg}
+                                        />
```

Comment:
isn't it that validation messages can be array of object(message, message_status) ?
example:
```
[
    { message: "Minimum 8 characters", status: "success"},
    { message: "At least 1 uppercase letter", status: "error"},
]
```